src/mesh/parallel.jl

```diff
@@ -1,5 +1,8 @@
-# Partition mesh using a static domain decomposition algorithm based on leaf cell count alone
-function partition!(mesh::ParallelTreeMesh)
+# Partition mesh using a static domain decomposition algorithm
+# based on leaf cell count and tree structure.
+# If all children of a cell are leaves, the algorithm will keep them together on one rank
+# to allow for coarsening.
+function partition!(mesh)
```

Comment:
Why did you remove the `::ParallelTreeMesh`?

Reply:
By mistake. I just copied the old version and didn't realize that this has changed while switching to Taal.